src/python/dxpy/utils/app_unbuilder.py

```diff
@@ -94,7 +110,8 @@ def dump_executable(executable, destination_directory, omit_resources=False, des
                     handler_id = handler.get_id()
                     fname = "resources/%s.tar.gz" % (handler_id)
                     download_dxfile(handler_id, fname)
-                    subprocess.check_call(["tar", "-C", "resources", "-zxvf", fname], shell=False)
+                    print("Unpacking resources")
+                    subprocess.check_call(["tar", "-C", "resources", "-zxf", fname], stdout=subprocess.PIPE, shell=False)
```

Comment:
Was the idea just to suppress the "tar" output here? According to the documentation, using PIPE in conjunction with check_call should be avoided because it can deadlock the process (since you have no mechanism to consume stdout while the program is running). I think you can omit this since "tar" shouldn't be printing anything in normal operation anyway.